src/main/resources/lang/french.lang

```diff
@@ -0,0 +1,190 @@
+		help: Affiche ce message d'aide. Utilisez '/skript reload/enable/disable/update' pour obtenir plus d'informations
+		reload:
+			description: Recharge la configuration, tous les scripts, tout, ou un script spécifique
+			all: Recharge la configuration, toutes les configurations d'alias et tous les scripts
+			config: Recharge la configuration principale
+			aliases: Recharge les configurations d'alias (aliases-english.zip ou jar du plugin)
+			scripts: Recharge tous les scripts
+			<script>: Recharge un script spécifique ou un dossier de scripts
+		enable:
+			description: Active tous les scripts ou un script spécifique
+			all: Active tous les scripts
+			<script>: Active un script spécifique ou un dossier de scripts
+		disable:
+			description: Désactive tous les scripts ou un script spécifique
+			all: Désactive tous les scripts
+			<script>: Désactive un script spécifique ou un dossier de scripts
+		update:
+			description: Vérifie les mises à jour, liste le journal des modifications ou télécharge la dernière version de Skript
+			check: Vérifie la présence d'une nouvelle version
+			changes: Liste toutes les modifications apportées depuis la version actuelle
+			download: Télécharge la dernière version
+		info: Affiche un message contenant les liens vers les alias et la documentation de Skript
+		gen-docs: Génère la documentation en utilisant doc-templates dans le dossier du plugin
+		test: Utilisé pour exécuter les tests Skript
+
+	invalid script: Impossible de trouver le script <grey>'<gold>%s<grey>'<red> dans le dossier des scripts !
+	invalid folder: Impossible de trouver le dossier <grey>'<gold>%s<grey>'<red> dans le dossier des scripts !
+	reload:
+		warning line info: <gold><bold>Ligne %s :<gray> (%s)<reset>\n
+		error line info: <light red><bold>Ligne %s :<gray> (%s)<reset>\n
+		reloading: Rechargement de <gold>%s<reset>...
+		reloaded: <gold>%s<lime> rechargé(s) avec succès. <gray>(<gold>%2$sms<gray>)
+		error: <gold>%2$s <light red>¦erreur rencontrée¦erreurs rencontrées¦ lors du rechargement de <gold>%1$s<light red>! <gray>(<gold>%3$sms<gray>)
+		script disabled: <gold>%s<reset> est actuellement désactivé. Utilisez <gray>/<gold>skript <cyan>enable <red>%s<reset> pour l'activer.
+		warning details: <yellow>    %s<reset>\n
+		error details: <light red>    %s<reset>\n
+		other details: <white>    %s<reset>\n
+		line details: <gold>    Ligne : <gray>%s<reset>\n <reset>
+
+		config, aliases and scripts: la configuration, les alias et tous les scripts
+		scripts: tous les scripts
+		main config: la configuration principale
+		aliases: les alias
+		script: <gold>%s<reset>
+		scripts in folder: tous les scripts dans <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <reset>script¦¦s¦ dans <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> ne contient aucun script activé.
+	enable:
+		all:
+			enabling: Activation de tous les scripts désactivés...
+			enabled: Activation et analysation réussis de tous les scripts précédemment désactivés.
```

Comment:
The word `analysation `does not exist in French
Perhaps it should be replaced by something like `analyse`

Reply:
Oops, that was frenglish 😂 Thanks for catching it!